Assets/MixedRealityToolkit/_Core/Definitions/Devices/InteractionDefinition.cs

```diff
@@ -20,6 +20,14 @@ public InteractionDefinition(uint id, AxisType axisType, DeviceInputType inputTy
             InputType = inputType;
         }
 
+        public InteractionDefinition(uint id, AxisType axisType, Devices.DeviceInputType inputType, InputAction inputAction) : this()
```

Comment:
We don't need an overload.

Reply:
I do, to create the interaction for the controller

Reply:
We will most likely always require an action, correct?